### ISSUE TITLE
Action based on Dependabot Alerts?

### ISSUE THREAD

**Issue by klaus-halfmann** [widelands/widelands-website#429]
I get Alerts from Dependabot every Day since a while.
As we use the same Tool in my Job its only fair to
call for actions on such Alerts also here.

Quite some automatically created MRs can be found here:
https://github.com/widelands/bunnybot/security/dependabot

Is it safe to just "click them trough"? Like in https://github.com/widelands/bunnybot/security/dependabot/18 ?

**Comment by frankystone** [comment]
> Is it safe to just "click them trough"? 

No, updating packages should at least be tested before deploying.
